seq2science/rules/peak_counts.smk

```diff
@@ -1,48 +1,288 @@
-def get_peak_replicates(wildcards):
+ruleorder: macs2_callpeak > narrowpeak_summit
+
+
+def count_table_output():
+    """
+    get all the count table outputs.
+    """
+    ftype = get_ftype(list(config['peak_caller'].keys())[0])
+    if ftype != "narrowPeak":
+        return []
+
+    print(expand(["{result_dir}/count_table/{peak_caller}/{assemblies}_{normalization}.tsv"],
```

Comment:
smart testing method. but can be removed ofcourse.

additionally, you are `expand`ing with the config, so could you not replace all those f"...{config['logbase]" with "...{logbase}"?

Reply:
Ouch, I missed that print! It's huge, difficult to miss :blush: 

No this can not be replaced, unfortunately. There are two string "interpolations" happening at the same time. Once in the normalization kwargs, and once in the path string. :(

Reply:
ah OK, and an expand(expand()) isn't possible I guess?

Reply:
Probably possible, but then you would have to do some curly braces magic which I think is a worse solution for the next person touching that... :thinking: